src/Symfony/Component/Validator/Constraints/ImageValidator.php

```diff
@@ -234,4 +238,29 @@ public function validate(mixed $value, Constraint $constraint)
             imagedestroy($resource);
         }
     }
+
+    /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    private function isSvg($value)
```

Comment:
Please use native type declarations. Can we be more precise about the `$value` type?

Reply:
$value can be an instance of File or string. that's way I put mixed